store/todos.js

```diff
@@ -1,26 +1,44 @@
 import { uuid } from 'uuidv4';
-import { Limit } from '@/constants.js';
+import { forEach } from 'lodash';
+import { COMPLETE_TASK } from '../constants';
+import {
+  LIMIT,
+  // eslint-disable-next-line camelcase
+  All_Task,
+  // eslint-disable-next-line camelcase
+  Incomplete_Task,
+  // eslint-disable-next-line camelcase
+  Complete_Task,
```

Comment:
please remove eslint rule disable for these lines and use UPPERCASE letters.

Reply:
Resolved